requirements.txt

```
@@ -15,5 +15,6 @@ toml
 
 # tools
 mapfile-parser>=2.3.5,<3.0.0
+pyelftools>=0.30,<1.0.0
```

```suggestion
pyelftools==0.30
```
afaik pyelftools doesn't use semver, and I wouldn't trust for it to not have accidentally breaking changes at any point, may as well just pin a version